packages/chart/src/ChartUtils.ts

```diff
@@ -681,17 +681,22 @@ class ChartUtils {
    * Converts the Iris plot style into a plotly chart type
    * @param plotStyle The plotStyle to use, see dh.plot.SeriesPlotStyle
    * @param isBusinessTime If the plot is using business time for an axis
+   * @param allowWebGL If WebGL is allowedd
    */
   getPlotlyChartType(
     plotStyle: DhType.plot.SeriesPlotStyle,
-    isBusinessTime: boolean
+    isBusinessTime: boolean,
+    allowWebGL: boolean
```

Comment:
Should this be mentioned as a BREAKING CHANGE?

Reply:
I'll just default it to `true` to avoid making it a breaking change. I wanted to make sure I caught all the places where it was called from though.